README.md

```diff
@@ -36,7 +36,7 @@
       API Docs
     </a>
     <span> | </span>
-    <a href="https://github.com/rustasync/tide/blob/master/.github/CONTRIBUTING.md">
+    <a href="./.github/CONTRIBUTING.md">
```

Comment:
The changes here are remnants of old commits from #220, you may want to keep this in mind later when fixing the merge conflicts.

(long paths are required so the URLs on crates.io can be fully resolved)

Reply:
Yeah, will likely have to rebase with master again once it gets merged.